codegen/src/main/java/io/sundr/codegen/processor/JavaGeneratingProcessor.java

```diff
@@ -207,4 +207,9 @@ private static boolean classExists(TypeDef typeDef) {
       return false;
     }
   }
+
+  private void printSkipping(String name, String msg) {
+    processingEnv.getMessager().printMessage(Diagnostic.Kind.OTHER,
```

Comment:
@iocanel is the "Skipping" message a warning or info?

Reply:
Info or debug